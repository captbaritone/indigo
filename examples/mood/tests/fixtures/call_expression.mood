fn add(a: f64, b: f64): f64 {
  a + b
}

fn double(a: f64): i32 {
  add(a, a)
}

pub fn test(): i32 {
  double(5)
}