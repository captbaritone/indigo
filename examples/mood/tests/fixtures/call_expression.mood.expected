Error: Expected i32, got f64:
 --> call_expression.mood:6:3

  | fn double(a: f64): i32 {
6 |   add(a, a)
  |   ^^^^^^^^^ This expression has the wrong type.
  | }
